packages/block-library/src/cover/edit.js

```diff
@@ -223,14 +246,29 @@ class CoverEdit extends Component {
 									step={ 10 }
 									required
 								/>
-							</PanelColorSettings>
-						</PanelBody>
-					</InspectorControls>
-				) }
+							) }
+						</PanelColorSettings>
+					) }
+				</InspectorControls>
 			</>
 		);
 
-		if ( ! url ) {
+		function StatefulColorPalette( props ) {
```

Comment:
We have a color palette component part of block editor that automatically adds the current theme colors to the pallet: https://github.com/WordPress/gutenberg//blob/359858da0675943d8a759a0a7c03e7b3846536f5/packages/block-editor/src/components/color-palette/index.js

I guess we can use it here and avoid the need to query colors at the block level.

Reply:
👍🏻